code/modules/surgery/organs/internal/liver/_liver.dm

```diff
@@ -266,21 +276,23 @@
 	name = "upgraded cybernetic liver"
 	desc = "An upgraded version of the cybernetic liver, designed to improve further upon organic livers. It is resistant to alcohol poisoning and is very robust at filtering toxins."
 	icon_state = "liver-c-u2"
-	alcohol_tolerance = 0.001
+	alcohol_tolerance = ALCOHOL_RATE * 0.2
 	maxHealth = 2 * STANDARD_ORGAN_THRESHOLD
 	toxTolerance = 10 //can shrug off up to 10u of toxins
 	liver_resistance = 1.5 * LIVER_DEFAULT_TOX_RESISTANCE // +50%
 	emp_vulnerability = 20
 
-/obj/item/organ/internal/liver/cybernetic/emp_act(severity)
-	. = ..()
-	if(. & EMP_PROTECT_SELF)
-		return
-	if(!COOLDOWN_FINISHED(src, severe_cooldown)) //So we cant just spam emp to kill people.
-		owner.adjustToxLoss(10)
-		COOLDOWN_START(src, severe_cooldown, 10 SECONDS)
-	if(prob(emp_vulnerability/severity)) //Chance of permanent effects
-		organ_flags |= ORGAN_EMP //Starts organ faliure - gonna need replacing soon.
+/obj/item/organ/internal/liver/cybernetic/surplus
+	name = "surplus cybernetic liver"
+	desc = "A very cheap cybernetic liver, designed to be mass produced. \
+		It is very fragile and absolutely terrible at filtering toxins. \
+		Offers no protection against EMPs."
+	icon_state = "liver-c-s"
+	maxHealth = STANDARD_ORGAN_THRESHOLD * 0.35
```

Comment:
Out of scope but these should probably be standardized between organ tiers

Reply:
Also, have you considered making these available roundstart for doctors as ghetto organ replacements

Reply:
I'll do it in a separate PR.